packages/amplify-migration-template-gen/src/cfn-stack-updater.ts

```diff
@@ -0,0 +1,70 @@
+import { CloudFormationClient, DescribeStacksCommand, Parameter, UpdateStackCommand } from '@aws-sdk/client-cloudformation';
+import { CFNTemplate } from './types';
+import assert from 'node:assert';
+
+const POLL_ATTEMPTS = 60;
+const POLL_INTERVAL_MS = 1500;
+const NO_UPDATES_MESSAGE = 'No updates are to be performed';
+const CFN_IAM_CAPABILIY = 'CAPABILITY_NAMED_IAM';
+const COMPLETION_STATE = '_COMPLETE';
+export const UPDATE_COMPLETE = 'UPDATE_COMPLETE';
+/**
+ * Updates a stack with given template. If no updates are present, it no-ops.
+ * @param cfnClient
+ * @param stackName
+ * @param parameters
+ * @param templateBody
+ * @param attempts number of attempts to poll CFN stack for update completion state. The interval between the polls is 1 second.
```

Comment:
Nit: `POLL_INTERVAL_MS` is set to 1.5 seconds and the comments mention interval as 1 second. Is the context different for the constant defined and the parameter here?

Reply:
Good catch! Will update the comment 🙂